TestAssets/TestProjects/StandaloneApp/project.json

```diff
@@ -0,0 +1,42 @@
+{
+  "buildOptions": {
+    "emitEntryPoint": true
+  },
+  "frameworks": {
+    "netcoreapp1.0": {
+      "dependencies": {
+        "Microsoft.NETCore.App": "1.0.0-rc3-*"
+      }
+    }
+  },
+  "runtimes": {
+    "win7-x64": {},
+    "win7-x86": {},
+    "win8-x64": {},
```

Comment:
We need every platform we're going to test on. I wasn't able to do a `dotnet build --runtime win10-x64` without this and I think we should stay away from hacks like `if windows use win7-x64` where possible.